File: program/SESI4/case.pas
var
   nilai : char;
begin
   readln(nilai);

   case (nilai) of
      'A' : begin writeln('Sempurna'); end;
      'B' : begin writeln('Bagus'); end;
      'C' : begin writeln('Cukup'); end;
      'D' : begin writeln('Kurang'); end;
      'E' : begin writeln('Sangat kurang'); end;
   end;
end.

File: topik/pemrograman-dasar-cpp/05-percabangan/asset/case.cpp
#include <cstdio>

char buff[1001];

int main() {
  scanf("%s", buff);
  char nilai = buff[0];

  switch (nilai) {
    case 'A':
      printf("Sempurna\n");
      break;

    case 'B':
      printf("Bagus\n");
      break;

    case 'C':
      printf("Cukup\n");
      break;

    case 'D':
      printf("Kurang\n");
      break;
  }
}

